src/main/java/com/mg105/entities/Doorway.java

```diff
@@ -1,5 +1,6 @@
 package com.mg105.entities;
 
+public class Doorway {
```

Comment:
Thanks for adding the header for class, I believe I just wrongly delete this when merge the conflict in Doorway.